runtime/manifest-parser.peg

```diff
@@ -90,8 +90,10 @@ Particle
       } else if (item.kind == 'description') {
         description = {};
         item.description.forEach(d => { description[d.name] = d.pattern; });
-      } else {
+      } else if (item.affordance) {
         affordance.push(item.affordance)
+      } else {
+        error('Unknown particle element ', item);
```

Comment:
maybe something like `particle description name ${item.name} doesn't match particle name ${name}` to be a bit more descriptive?